R/fit.R

```diff
@@ -1050,6 +1083,9 @@ CmdStanGQ <- R6::R6Class(
   private = list(
     # inherits draws_ and metadata_ slots from CmdStanFit
     read_csv_ = function(variables = NULL) {
+      if (!length(self$output_files(include_failed = FALSE))) {
+        stop("Generating quantities for all input MCMC chains failed. Unable to retrieve the generated quantities.")
```

Comment:
These errors will never happen right now, as we would never even call read_csv if no chain finished. Still think we can add them so if we ever add some other use of read_csv we have this case covered.

Reply:
That makes sense. Can you remove the word `"input"` from the message? It's not wrong, but I think we should use the same message as in the draws() method above (line 1054).